docs/source/algorithms/CorelliPowderCalibrationDatabase-v1.rst

```diff
@@ -0,0 +1,145 @@
+
+```
+20201025, 0.0001, -0.0002, 0.003, 0, 0, 0, 0
+```
+
+and the following lines to *corelli_bank001.csv*:
+20201025, 0.9678, 0.0056, 0.0003, 0.4563, -0.9999, 0.3424, 5.67
+
+The header for files *corelli_source.csv*, *corelli_sample.csv*, and files *corelli_bankXXX.csv* should be:
+```
+# YYYMMDD, Xposition, Yposition, Zposition, XdirectionCosine, YdirectionCosine, ZdirectionCosine, RotationAngle
+```
+
+**Combined Calibrtion Database File**: The last line of *corelli_source.csv*, *corelli_sample.csv*, and each of existing files *corelli_bankXXX.csv* will be collected and gathered into file *corelli_instrument_20202015.csv*. The header for this file should be:
+```
+# ComponentName, Xposition, Yposition, Zposition, XdirectionCosine, YdirectionCosine, ZdirectionCosine, RotationAngle
+```
+The name of the file should be output in the workbench logger.
+
+
+Usage
+-----
+..  Try not to use files in your examples, 
+    but if you cannot avoid it then the (small) files must be added to 
+    autotestdata\UsageData and the following tag unindented
+    .. include:: ../usagedata-note.txt
+
+**Example - CorelliPowderCalibrateDatabase**
+
+.. testcode:: CorelliPowderCalibrationExample
+    
+    # Import modules
+    import numpy as np
+    import os
+    
+    # Enpty workspace
+    input = LoadEmptyInstrument(InstrumentName='CORELLI')
+    # add start timedelta64
+    AddSampleLog(Workspace=input, LogName='start_time', LogText='2020-02-20T12:57:17', LogType='String')
+    
+    # generate simulated stored database files
+    bank2_str = "# YYYMMDD , Xposition , Yposition , Zposition , XdirectionCosine , YdirectionCosine , ZdirectionCosine , RotationAngle\n" \
+                "# str , double , double , double , double , double , double , double\n" \
+                "20001117,0.0001,-0.0002,0.003,0,-23.3,98.02,0"
+    
+    # generate simulated stored database files
+    bank12_str = "# YYYMMDD , Xposition , Yposition , Zposition , XdirectionCosine , YdirectionCosine , ZdirectionCosine , RotationAngle\n" \
+                "# str , double , double , double , double , double , double , double\n" \
+                "20011117,1.0001,-2.0002,3.003,4,-23.3,98.02,0"
+    
+    calib_dir = 'sim_corelli_cal'
+    if not os.path.exists(calib_dir):
+        os.mkdir(calib_dir)
+    
+    for bank, content in [('bank2', bank2_str), ('bank12', bank12_str)]:
+        bankfile = open(os.path.join(calib_dir, bank + '.csv'), 'w')
+        bankfile.write(content)
+        bankfile.close()
+    
+    # Create table
+    calib_table = CreateEmptyTableWorkspace("CorelliCalibrationTestTable");
+    calib_table.addColumn("str", "ComponentName")
+    for colname in ["Xposition", "Yposition", "Zposition","XdirectionCosine", "YdirectionCosine", "ZdirectionCosine", "RotationAngle"]:
+        calib_table.addColumn("double", colname)
+    
+    # add entry
+    calib_table.addRow(["source" , 0. , 0. , -15.560 , 0. , 0. , 0., 0.])
```

Comment:
to be consistent, can you use CORELLI's component names for source and sample ("moderator" and "sample-position")?